File: xdef.bridge/xdef.bridge/src/main/java/org/xdef/bridge/Server/Requests/RemoteCallException.java
package org.xdef.bridge.server.requests;

public class RemoteCallException extends Exception{

    private static final long serialVersionUID = 3051187634212893330L;
    private int errorCode;
    
    public RemoteCallException(int errorCode, String message) {
        super(message);
        this.errorCode = errorCode;
        
    }

    public int getErrorCode() {
        return errorCode;
    }
    
}

File: xdef.net/xdef.net/Connection/RemoteCallException.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace xdef.net.Connection
{
    public class RemoteCallException : Exception
    {
        public int ErrorCode { get; }

        public RemoteCallException(int errorCode, string message) : base(message)
        {
            ErrorCode = errorCode;
        }

        public RemoteCallException(string message, Exception innerException) : base(message, innerException)
        {
        }
    }
}
